oneandone/src/main/java/org/apache/jclouds/oneandone/rest/domain/Location.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.jclouds.oneandone.rest.domain;
+
+public enum Location {
+
+   DE("de", "Germany"),
+   US("us", "USA"),
+   ES("es", "Spain"),
+   GB("gb", "UNITED KINGDOM"),
+   UNRECOGNIZED("unrecognized", "Unrecognized location"),
+   MOCK("mock", "Mock");

```

Comment:
Let's use mock tests with real values but avoid putting test code in the production code.